client/src/app/site/pages/organization/pages/committees/committees.subscription.ts

```diff
@@ -8,6 +8,9 @@ import { ViewMeeting } from '../../../meetings/view-models/view-meeting';
 import { ViewCommittee } from './view-models';
 
 export const COMMITTEE_LIST_SUBSCRIPTION = `committee_list`;
+export const COMMITTEE_LIST_MINIMAL_SUBSCRIPTION = `committee_list`;
```

Comment:
```suggestion
export const COMMITTEE_LIST_MINIMAL_SUBSCRIPTION = `committee_list_minimal`;
```

You could correct this while you are at it.